app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -283,6 +283,9 @@ const helpContents: { [key: string]: string } = {
   'pipeline.config.bake.manifest.overrideExpressionEvaluation':
     '<p>Explicitly evaluate SpEL expressions in overrides just prior to manifest baking. Can be paired with the "Skip SpEL evaluation" option in the Deploy Manifest stage when baking a third-party manifest artifact with expressions not meant for Spinnaker to evaluate as SpEL.</p>',
   'pipeline.config.bake.manifest.templateRenderer': '<p>This is the engine used for rendering your manifest.</p>',
+  'pipeline.config.bake.manifest.helm.chartFilePath': `
+    <p>This is the relative path to the Chart.yaml file within your Git repo.</p>
+    <p>e.g.: <b>helm/my-chart/Chart.yaml</b></p>`,
```

Comment:
I think a lot of places use `<code>` tags for something like this (i.e., instead of the `<b>` tag), but whatever looks best.

Reply:
I did the same thing as `kustomize.filePath` below...trying my best to not re-invent the wheel here.